java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
Thanks, Bob. I think the MessageFormat class Randall referred to can potentially do much more than simple String concatenation.
The latter I am happy to use. What I didn’t know was the proper term for the ``{0}`` part (hint: “variable”, seems logical).
PS: I’ll leave the MessageFormat number formatting tricks to the advanced members ;-)

> Op 20 aug. 2016, om 18:00 heeft Bob Jacobsen <notifications@github.com> het volgende geschreven:
> There are properties strings with '{..}' in e.g. java/src//jmri/jmrix/JmrixBundle.properties, java/src//jmri/jmrit/Bundle.properties, java/src//jmri/implementation/ImplementationBundle.properties, even java/src//jmri/NamedBeanBundle.properties. You can pick one of those and search the code to find how it's used. For example, from NamedBeanBundle.properties (…)
>